### DISCUSSION TITLE
easiest "repeat last"

### DISCUSSION THREAD

**Discussion by soloturn** [helix-editor/helix#10661] [General]
i saw a couple of "repeat" tickets, but was not able to figure out a useful strategy to "repeat last". i am using the repeat in 3 cases:
* search something, repeat action
* navigate somewhere and repeat action

i tried multiple cursors, i am not able to remember the key sequence. this would at least solve the search case. how do you guys approach this repeat issue?

**Comment by TornaxO7** [comment]
I think this should be converted into a discussion.

However, my approach would be probably to use macros here. In general: If it gets more complex => Macros

**Reply by soloturn** [reply]
a macro for repeating the last tihng? i tried it - but this is kind of complex to type. or how you do it?
